apps/yggdrasil/lib/yggdrasil/role_permission.ex

```diff
@@ -0,0 +1,13 @@
+defmodule Yggdrasil.RolePermission do
+  use Ecto.Schema
+
+  @primary_key false
+  schema "roles_permissions" do
+    field :resource, :string
+    field :permission, :string
```

Comment:
@copenhas so I don't think there is a need to look up that actual resource/permission record per your comment earlier. So I dropped the `belongs_to` here, and just made them normal fields.

The db like you pointed out will ensure the correctness of these values on insert, but it simplifies the code a bit this way.

I think this reads better.